addons/web/static/src/webclient/webclient.scss

```diff
@@ -117,7 +117,11 @@ div.o_boolean_toggle.custom-control.custom-checkbox {
 
 //== Badges
 .badge {
-  margin: 1px 2px 1px 0;
+  min-width: $o-badge-min-width;
+
+  + .badge {
+    margin-inline-start: 2px;
+  }
```

Comment:
I don't like this kind of magic, why not use the much more explicit `ms-2` on the following elements that need it?

Reply:
Using a class (`.ms-1` = 3px) it's feasible, but It would imply to dynamically toggle these classes in m2m fields.

The problem is that these fields has not been converted to OWL yet. Chances are the the layout will change and for that time maybe we could already take advantage of new BTS5 classes ( eg. use a grid layout with gaps).

So I do agree with you, but I'd suggest to keep the "magic" part for now: the margin applied is very subtle (less than a `m-1`) and what does is just to add a minor visual separation, It's very unlikely that you want two badges attached to each other anyways... 

btw, I now realised that I forgot to handle vertical margin, I'll fix that